app/presenters/content_change_presenter.rb

```diff
@@ -0,0 +1,45 @@
+class ContentChangePresenter
+  EMAIL_DATE_FORMAT = "%I:%M %P on %-d %B %Y".freeze
+
+  def initialize(content_change)
+    @content_change = content_change
+  end
+
+  def self.call(content_change)
```

Comment:
Should we use `*args` here for consistency with the other classes, and in case we decide to change the method signature?

Reply:
Fixed in 809c5d2